action.yml

```
@@ -6,8 +6,16 @@ branding:
   icon: 'settings'
   color: 'gray-dark'
 
+inputs:
+  directories:
+    description: 'Comma-separated list of directories to look for coverage information (e.g. `src,examples`)'
+    required: false
+    default: 'src'
+
 runs:
   using: 'composite'
   steps:
     - run: julia --color=yes "$GITHUB_ACTION_PATH"/main.jl
       shell: bash
+      env:
+        - INPUT_DIRECTORIES
```

maybe it should be
```suggestion
      env: INPUT_DIRECTORIES
```
?

---

From [here](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example):
> When you specify an input to an action in a workflow file or use a default input value, GitHub creates an environment variable for the input with the name INPUT_<VARIABLE_NAME>.

Thus maybe this should be just omitted?
```suggestion
```

---

https://github.com/julia-actions/julia-processcoverage/pull/8/files#r645527013